test/scripts/e2e-kind.sh

```diff
@@ -45,6 +45,27 @@ should set default value on new IngressClass
 should prevent Ingress creation if more than 1 IngressClass marked as default
 "
 
+IPV4_ONLY_TESTS="
+# shard-n Tests
+Network.+should resolve connrection reset issue
+\[Feature:Networking-IPv4\]
+
+# shard-np Tests
+NetworkPolicy.+should allow egress access to server in CIDR block
+
+# shard-s Tests
+Services.+should create endpoints for unready pods
```

Comment:
Why is this IPv4 only? All of the entries here should explain that, so that if the test changes in the future we can see that we should move them out of the ipv4-only list.

(Really, the upstream e2e tests should know to not run IPv4-only tests in single-stack IPv6 clusters.)

Reply:
You are right. I just pulled out the tests that consistently failed IPv6 only with the plan to come back and figure out why. Then got buried in the rest of the PR and never got back. I need to investigate.

To your second point, `\[Feature:Networking-IPv4\]` were not being skipped, so looks like upstream isn't skipping for IPv6, so need to dig into that code and see why.